cli/common/src/prometheus.rs

```diff
@@ -307,6 +326,91 @@ pub fn write_solido_metrics_as_prometheus<W: io::Write>(
     Ok(())
 }
 
+pub fn write_anker_metrics_as_prometheus<W: io::Write>(
+    metrics: &anker::metrics::Metrics,
+    at: SystemTime,
+    out: &mut W,
+) -> io::Result<()> {
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "anker_swapped_rewards_st_sol_total",
+            help: "Total amount of stSOL rewards swapped by our Anker instance.",
+            type_: "counter",
```

Comment:
```suggestion
            type_: "gauge",
```

This is data that comes from the chain, not from inside the daemon itself, right? In that case it should be a gauge, not a counter. The value might go down for example when the daemon connects to a loadbalanced RPC node, and one of the RPC nodes behind the loadbalancer is a few slots behind. Then we can get this flip-flopping behavior, and Prometheus should not treat that as a counter reset.